consensus/ethash/algorithm_test.go

```diff
@@ -702,7 +702,11 @@ func TestConcurrentDiskCacheGeneration(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Failed to create temporary cache dir: %v", err)
 	}
-	defer os.RemoveAll(cachedir)
+	defer func() {
+		if err := os.RemoveAll(cachedir); err != nil {
+			panic(err)
```

Comment:
This check is useless because the directory is temporary and it doesn't matter whether it can't be deleted.